src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionsLoaderTest.java

```diff
@@ -305,4 +307,48 @@ public void testSettingModuleId() throws Exception {
         final String id = Whitebox.getInternalState(suppressElement, "moduleId");
         assertEquals("Id has to be defined", "someId", id);
     }
+
+    @Test
+    public void testXpathSuppressions() throws Exception {
+        final String fn = getPath("suppressions_xpath_correct.xml");
+        final Set<TreeWalkerFilter> filterSet = SuppressionsLoader.loadXpathSuppressions(fn);
+
+        assertEquals("Invalid number of filters", 1, filterSet.size());
+    }
+
+    @Test
+    public void testXpathInvalidFileFormat() throws IOException {
+        final String fn = getPath("suppressions_xpath_invalid_file.xml");
+        try {
+            SuppressionsLoader.loadXpathSuppressions(fn);
```

Comment:
the test should fail if the exception is not thrown. Add fail() after  SuppressionsLoader.loadXpathSuppressions(fn);

Reply:
done